js/index.js

```diff
@@ -65,3 +65,47 @@ messageForm.addEventListener('submit', (e) => {
 
     messageForm.reset();
 });
+
+let githubRequest = new XMLHttpRequest();
+
+githubRequest.open("GET","https://api.github.com/users/up2code3/repos")
+githubRequest.send();
+
+addEventListener("load", (Event) => {
+  let repositories = JSON.parse(githubRequest.responseText);
+  console.log(repositories)
+
+
+// I compared the next to lines of code to the message section above to make sure it was right 
+let projectSection = document.getElementById("projects") 
+let projectList = projectSection.querySelector("ul") // this queries just the project section and not the whole page  
+
+
+for (var i = 0 ; i < repositories.length ; i++){
+  let project = document.createElement("li");  
+  project.innerText = repositories[i].name; 
+  projectList.appendChild(project);
+}
+})
+
+//  I was stuck on line 87 for awhile, it had 7 li under the 
+//  projects element on the page, but they all said object object,
+//  so eventually it dawned on me to do the .name to refer 
+//  to the key of the pair. but that didnt work, so i tired name:,
+//  i thought maybe the colon was aprat of the name, it through 
+//  an error, so I deleted the colon leaving it as .name
+//  and suddenly it worked... weird... , even as i was typing this 
+//  message all of the li's under projects disappeared, and 
+//  I thought i broke it again, then I did a refresh and they came back. 
+// this is the error message it threw when they all randomly disappear 
+// 
+//  VM2188:1 Uncaught SyntaxError: Unexpected end of JSON input
+//     at JSON.parse (<anonymous>)at index.js:75:27</anonymous>
```

Comment:
Something to keep in mind... This is telling you exactly where the code is experiencing an error: `index.js:75:27` means in index.js on line 75, 27 characters in. You can see you have a JSON.parse on that line which is something this error also mentioned. So, it's likely something was happening while fetching your data.

Keep in mind, when you are fetching data, it lives somewhere else. So, it may take a second for it to arrive if you have a slow connection.

I pulled your code down and it works! So that's good. Keep it up.